src/formats/AmberNetCDF.cpp

```diff
@@ -109,7 +109,14 @@ UnitCell AmberNetCDFFormat::read_cell() {
     assert(length.size() == 3);
     assert(angles.size() == 3);
 
-    return {length[0], length[1], length[2], angles[0], angles[1], angles[2]};
+    return {
+        static_cast<double>(length[0]),
```

Comment:
What is ``length``'s type here?  The auto above is misleading in this case.

Reply:
The type is `NcFloat`, a netcdf array of floats.